client/events.lua

```diff
@@ -129,20 +129,6 @@ RegisterNetEvent('QBCore:Command:GoToMarker', function()
 end)
 
 -- Vehicle Commands
-lib.callback.register('qbx_core:client:vehicleSpawned', function(netId, props)
-    local veh = NetworkGetEntityFromNetworkId(netId)
-
-    for i = -1, 0 do
-        local ped = GetPedInVehicleSeat(veh, i)
-        if ped ~= cache.ped and ped > 0 and NetworkGetEntityOwner(ped) == cache.playerId then
-            DeleteEntity(ped)
-        end
```

Comment:
Vehicles sometimes spawn with NPCs which this took care of deleting. Can we move this logic over as well?

Reply:
done